src/docbook-tool.js

```diff
@@ -0,0 +1,114 @@
+"use strict";
```

Comment:
Can we add a comment what this is for please? Ideally something to stop someone coming along later down the line who forgot what this thing is for and deleting it, thus breaking downstream systems

Reply:
_starts looking up why we put this in here_

Reply:
Not specifically `use strict` - I mean the whole file in general :grin:

Reply:
Oh, lol. That makes more sense.